source/docs/articles/sites/newrelic.md

```diff
@@ -12,6 +12,18 @@ Using New Relic on Pantheon not only makes it easy for you to monitor to your pe
 ##Enabling New Relic
 Enable New Relic on your Pantheon site by clicking **Settings** in the upper-right corner of your Site Dashboard. Go to the Add Ons tab, and click **Add** next to New Relic.
 ![New Relic Add On](/source/docs/assets/images/new-relic-add-on-image.png)
+
+## Remove Multidev Environments in New Relic
+The applications are added automatically, but after deletion, you'll need to manually [remove them in New Relic](https://docs.newrelic.com/docs/apm/new-relic-apm/maintenance/remove-applications-servers).  
+
+1. From your Dashboard, select the **New Relic** tab, and **Open New Relic**.  
+  * To delete an app: From the New Relic menu bar, select **APM** > **Applications**.  
+  * To delete a server: From the New Relic menu bar, select **Servers** > **Servers**.
+2. Wait until the color-coded health status to turn gray, then select the app's or server's gear icon.
+3. Select **Delete app** from New Relic APM or **Delete server** from New Relic Servers, and then select the confirmation button.
+
+![Remove New Relic](/source/docs/assets/images/remove-new-relic.png)
```

Comment:
Yes. The one you put in the issue doesn't show much of the screen and I can't get in to NR to grab another.